tests/test_logging.py

```diff
@@ -276,6 +288,20 @@ def test_formatter_large_msg(monkeypatch):
     assert structured == {}
 
 
+def test_formatter_protected(monkeypatch):
+    fmt = logs.StructuredFormatter()
+
+    # make this formatter error
+    def error():
```

Comment:
let's make this take an argument so it fails because of the `raise` rather than the wrong number of arguments

<3 LGTM for finding this.